packages/edit-site/src/hooks/commands/use-common-commands.js

```diff
@@ -288,6 +310,91 @@ function useGlobalStylesOpenRevisionsCommands() {
 		isEditorPage,
 		getCanvasMode,
 		setCanvasMode,
+		closeGeneralSidebar,
+	] );
+
+	return {
+		isLoading: false,
+		commands,
+	};
+}
+
+function useGlobalStylesStyleBookCommands() {
+	const {
+		openGeneralSidebar,
+		closeGeneralSidebar,
+		setEditorCanvasContainerView,
+		setCanvasMode,
+	} = unlock( useDispatch( editSiteStore ) );
+	const { getCanvasMode, getEditorCanvasContainerView } = unlock(
+		useSelect( editSiteStore )
+	);
```

Comment:
Similar to what was mentioned in `useGlobalStylesOpenRevisionsCommands`, it might be better to unlock it within the existing `useSelect` hook.